gitops/argocd/tekton-chains/overlays/openshift/kustomization.yaml

```diff
@@ -0,0 +1,39 @@
+---
+resources:
+  - ../../base
+
+patchesStrategicMerge:
```

Comment:
Why are the below 2 files under patchesStrategicMerge and not in resources?
Also, since you have base under resources, we don't need to load these explicitly no?

Reply:
Because they not defining new resources, but updating resources already defined in `resources`.
I do not understand your second comment.as those 2 patches are not in the `base` folder, but in the `overlays/openshift` folder. I'll let you resolve this conversation if you don't have any more questions, or follow up.